systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/resources/ResourceKind.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.messaginginfra.resources;
+
+public interface ResourceKind {
```

Comment:
I would consider if it wouldn't be better to use enom as long as we don't implement this interface and only using it as "static" call.